crates/turbopack-css/src/chunk/mod.rs

```diff
@@ -207,17 +207,32 @@ impl Asset for CssChunk {
 
         let path = self_vc.path();
         let chunk_name = path.to_string();
-        let mut code = format!("/* chunk {} */\n", chunk_name.await?);
 
-        let mut writer = WriterWithIndent::new(&mut code);
+        let mut body = "".to_owned();
+        let mut writer = WriterWithIndent::new(&mut body);
+        let mut external_imports = IndexSet::new();
         for entry in this.main_entries.await?.iter() {
             let entry_placeable = CssChunkPlaceableVc::cast_from(entry);
             let entry_content = entry_placeable.as_chunk_item(this.context).content();
 
-            expand_imports(&mut writer, entry_content).await?;
+            for external_import in expand_imports(&mut writer, entry_content).await? {
+                external_imports.insert(external_import.await?.to_owned());
+            }
         }
 
-        Ok(File::from(code).into())
+        Ok(File::from(
+            format!("/* chunk {} */\n", chunk_name.await?)
+                + &external_imports
+                    .iter()
+                    .map(|external_import| {
+                        format!("@import \"{}\";", external_import.replace('"', "\\\""))
```

Comment:
What's up with the replace?

Reply:
The result can't contain literal double quote characters as that would terminate the literal